vuepress/docs/next/tutorials/getting-started/eks-install.md

```diff
@@ -95,17 +95,17 @@ See [Creating an Amazon EKS Cluster](https://docs.aws.amazon.com/eks/latest/user
 ### Add the EBS CSI Add-on
 Starting with K8s 1.23, EKS requires an add-on in order to enable persistent volumes. The following instructions are for EBS CSI. See [the EBS CSI guide](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) and especially [the instructions for creating the EBS CSI IAM Role](https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html) for additional information.
 
-1. Determine the OpenID Connect provider URL for the cluster by going to `Clusters`→ YOUR-CLUSTER-NAME → `Overview`. It will be similar to this: 
+1. Determine the OpenID Connect provider URL for the cluster by going to `Clusters` → YOUR-CLUSTER-NAME → `Overview`. It will be similar to this: 
 ```
 https://oidc.eks.us-east-1.amazonaws.com/id/1C39B525EC0971750179719649SAMPLE
 ```
 
-2. Follow [the AWS instructions](https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html) to prepare the IAM role required for the EBS CSI Add-on to function correctly. You will need to setup a new Role, e.g. YOUR-EBS-ROLE, with an OpenID Connect Identity Provider for your cluster using the URL from the previous step.
+2. Follow [the AWS instructions](https://docs.aws.amazon.com/eks/latest/userguide/csi-iam-role.html) to prepare the IAM role required for the EBS CSI Add-on to function correctly. You will need to set up a new role, using an OpenID Connect Identity Provider based on the OIDC URL from the previous step.
```

Comment:
since the e.g. is taken out, the comma after role is not necessary>> You will need to set up a new role using an OpenID Connect Identity Provider based on the OIDC URL from the previous step.